sql/updates/characters/yyyy_mm_dd_xx_characters.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE `calendar_invites`
```

Comment:
These changes need to be in the character_base.sql file as well. Use mysqldump to update that file.